SegmentMesher/SegmentMesher.py

```diff
@@ -489,10 +417,11 @@ def createTempDirectory(self):
     qt.QDir().mkpath(dirPath)
     return dirPath
 
-  def createMeshFromSegmentationCleaver(self, inputSegmentation, outputMeshNode, additionalParameters = None, removeBackgroundMesh = False, paddingRatio = 0.10):
+  def createMeshFromSegmentationCleaver(self, inputSegmentation, outputMeshNode, segments = [], additionalParameters = None, removeBackgroundMesh = False, paddingRatio = 0.10, scale = 0.2, multiplier=0.5, grading=1.0):
```

Comment:
Parametrization has changed in Cleaver.  Parameters are now: feature scaling, sampling rate, and maximum rate of change